cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -8284,6 +8284,200 @@ spec:
       subresources:
         status: {}
 
+---
```

Comment:
These CRDs aren't in the pre-1.16/crds.yaml. Is it safe to assume we require 1.16+ for the volume replication controller?

Reply:
@Madhu-1 thoughts?

Reply:
I think we can go with volume replication is supported only in the kubernetes 1.16+ version (technically it should work with older versions also but we have tested with the latest kubernetes versions.